sssd_test_framework/roles/ipa.py

```diff
@@ -1103,6 +1105,64 @@ def passkey_remove(self, passkey_mapping: str) -> IPAUser:
         self._exec("remove-passkey", [passkey_mapping])
         return self
 
+    def vfido_passkey_add_register(
```

Comment:
I don't like that the method begins with `vfido` to differentiate it from the `umockdev` version. I was to propose to use method overloading, but apparently this functionality isn't available in python :facepalm: 

Can we manage this somehow so that the method is called the same way for `vfido` and `umockdev`?

Reply:
I'll look into that closer.  I did this here to be able to keep them separate while I was writing and testing the new one.

Reply:
Ok, that looks way better and this way we can continue providing the same method to external entities